### ISSUE TITLE
Can't clear vlan field

### ISSUE THREAD

**Issue by emil-nasso** [SpriteLink/NIPAP#1097]
I'm trying to clear the vlan field for a prefix in the web ui. The request that does away has an empty vlan field but the response still has the field filled in. It looks like everything is working on the page, no error messages and the field stays empty. If you reload the page, the vlan field is filled in again.

**Comment by plajjan** [comment]
Dupe of #1085 - with fix in #1092 !?

**Comment by emil-nasso** [comment]
@plajjan Ah, sounds great. I didn't get an error message at all though. It does sound like the PR would fix the issue though!

**Comment by garberg** [comment]
@emil-nasso can you confirm that this issue is resolved? :)

**Comment by emil-nasso** [comment]
No, sorry. I get an error now. The request contains the vlan field as an empty string. The response looks like this:

```
{"message": ["Unhandled data error."], "type": "NipapError", "error": 1}
```

This is how the form looks prior to clearing the vlan field and clicking save. 

![https://dl.dropboxusercontent.com/s/lznmhr651dbpuhs/2017-02-24%20at%2008.42.png](https://dl.dropboxusercontent.com/s/lznmhr651dbpuhs/2017-02-24%20at%2008.42.png)
